### DISCUSSION TITLE
MacBook display dims unexpectedly, after a while

### DISCUSSION THREAD

**Discussion by ymshulman** [MonitorControl/MonitorControl#929] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

MacBook display dims unexpectedly. Can't figure out when it happens; but it happens consistently during any working session of over 30 minutes. 

### Steps to reproduce

1. Plug in external monitor. Work on external monitor. 
(Seems to reproduce more quickly if I'm working on the external monitor than if I'm going back and forth between the two; but it does happen both ways.)

### Expected behavior

Display should only dim if I tell it to. I have not touched the keyboard buttons or any software sliders to make this happen. 

### Anything else?

n/a

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.0.1 
- Mac model: Macbook Pro (15-inch, 2018) 
- MonitorControl version: v4.0.2 
- Monitor(s): LG ultrawide 29" (2560x1080) 
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
To be honest I have no idea what's causing this. :) whenever it happens, it would be great to see al `.info` level log entries for MonitorControl (via the Console). MC should not change the brightness by itself. Are you sure this does not happen when MonitorControl is not running (maybe a faulty amibent light sensor?)

(check this https://github.com/MonitorControl/MonitorControl/discussions/851 out for help on how to see the logs)

**Comment by ymshulman** [comment]
I’ll try to grab those logs. But yes - I’m sure this doesn’t happen when
MonitorControl is not running, and has never happened before I installed
it.

**Comment by erikblomqvist** [comment, marked as answer]
@ymshulman I actually stumbled upon this problem as well, but I think I know what caused it and how to fix it.

If you go to **System Preferences > Displays > Display Settings…** and select the Built-in Retina Display in the left-hand sidebar, you just uncheck **Automatically adjust brightness**. Haven't had the problem since.
